provider/labels.go

```diff
@@ -15,18 +17,24 @@ import (
 // tfbridge.SchemaInfo.
 func fixLabelNames(prov *tfbridge.ProviderInfo) {
 	// Recursively applies the label fix
-	var apply func(string, shim.Schema, *tfbridge.SchemaInfo)
-	apply = func(key string, m shim.Schema, info *tfbridge.SchemaInfo) {
+	var apply func(string, shim.Schema, *tfbridge.SchemaInfo, *[]resource.PropertyPath, resource.PropertyPath)
+
+	apply = func(key string, m shim.Schema, info *tfbridge.SchemaInfo,
+		paths *[]resource.PropertyPath,
+		currentPath resource.PropertyPath,
+	) {
 		if key == "terraform_labels" {
 			info.Name = "pulumiLabels"
+			// Gather all paths to pulumiLabels
```

Comment:
As I said later in the review, `resource.PropertyPath.{Get,Set}` don't work correctly with globs, and right now they don't need to. To avoid shipping bugs to customers, lets assert here (during `make tfgen` time), that there are no globs in this path.
```suggestion
			for _, v := range currentPath {
				if v, ok := v.(string); ok && v == "*" {
					panic(fmt.Sprintf("at path %s: globs are not supported", currentPath))
				}
			}
			// Gather all paths to pulumiLabels
```